### PR TITLE
Terms of service

### PR DESCRIPTION
**Pull request by mbouzi**
- based on https://docs.google.com/document/d/1PMsVdqz_q0Sg1lDJzZsmObG3JMuPNZs5zZLSUn-AXuI/edit#heading=h.m6nrv52slvik
- located at "/terms-of-service"

run ```hugo server -D``` in root

### CODE REVIEW

File: content/terms-of-service/termsOfService.md [hunk 1]
```diff
@@ -1,3 +1,494 @@
...
+---
+
+<p>THESE TARDIGRADE TERMS OF SERVICE (THE “<b>AGREEMENT</b>”) ARE A LEGAL AGREEMENT BETWEEN YOU OR THE ENTITY YOU REPRESENT (“<b>USER</b>,” OR “<b>SUBSCRIBER</b>,” OR “<b>YOU</b>” OR “<b>YOUR</b>”) AND STORJ LABS INTERNATIONAL, SEZC,, INCLUDING ITS PARENT, SUBSIDIARIES, AFFILIATES, DIRECTORS, OFFICERS, EMPLOYEES, AGENTS, LICENSEES, AND EMPLOYEES (“<b>COMPANY</b>,” AS DEFINED BELOW, “<b>US</b>,” “<b>WE</b>” OR “<b>OUR</b>”) GOVERNING THE TERMS OF YOUR  USE OF THE STORAGE SERVICES AS DEFINED HEREIN.  THIS AGREEMENT HEREBY INCORPORATES BY REFERENCE THE FOLLOWING AGREEMENTS: <a target="_blank" href="" class="link">PRIVACY POLICY</a>, <a target="_blank" href="" class="link">TERMS OF USE</a>, <a target="_blank" href="" class="link">CODE OF CONDUCT</a>, AND <a target="_blank" href="" class="link">STORAGE NODE OPERATOR TERMS AND CONDITIONS.</a>  BY ACCEPTING THE TERMS OF THIS AGREEMENT, EITHER BY CLICKING TO SIGNIFY ACCEPTANCE, BY SETTING UP A USER ACCOUNT, OR BY USING THE STORAGE SERVICES, YOU AGREE TO BE BOUND BY THE PROVISIONS OF THIS AGREEMENT.   YOU REPRESENT AND WARRANT THAT YOU HAVE THE AUTHORITY TO ENTER INTO THIS AGREEMENT ON USER’S BEHALF. </p>
```
Comment (boshevski) [inline content/terms-of-service/termsOfService.md]: Missing the links url here.

File: content/terms-of-service/termsOfService.md [hunk 2]
```diff
@@ -1,3 +1,494 @@
...
+<br>
+<p><b>PLEASE READ THE TERMS OF THIS AGREEMENT CAREFULLY TO ENSURE THAT YOU UNDERSTAND EACH PROVISION. THIS AGREEMENT CONTAINS A BINDING ARBITRATION AND WAIVER OF CLASS ACTION RIGHTS PROVISION THAT REQUIRES THE USE OF INDIVIDUAL ARBITRATION TO RESOLVE DISPUTES AS SET FORTH BELOW.  IF YOU DO NOT AGREE TO THE TERMS OF THIS AGREEMENT, PLEASE DO NOT ACCESS OR USE THE STORAGE SERVICES.</b></p>
+<br>
+<p><b>IMPORTANT: THERE IS NO GUARANTEE THAT ANY MATERIALS STORED VIA THE STORAGE  SERVICES WILL BE AVAILABLE ON DEMAND OR RETRIEVABLE AT ANY TIME.  YOU ARE RESPONSIBLE FOR MAINTAINING BACKUP COPIES OF ALL STORAGE MATERIALS AT ALL TIMES. STORJ IS NOT RESPONSIBLE FOR ANY LOSS OR DAMAGE TO STORAGE MATERIALS.</b></p>
+<br>
+<p>1.&nbsp;&nbsp;<u>Definitions</u>.  For purposes of this Policy, the following terms have the meanings set forth below.</p>
```
Comment (boshevski) [inline content/terms-of-service/termsOfService.md:12]: "1. Definitions" - bold not underline

File: content/terms-of-service/termsOfService.md [hunk 3]
```diff
@@ -1,3 +1,494 @@
...
+							</ol>
+						</li>
+					</ol>
+				</li>
+			</ol>
+		</li>
+	</ol>
+</p>
+
+<p><b>10.&nbsp;&nbsp;Feedback</b>To the extent that User or its End Users provide Company with any suggestions, ideas, enhancement requests, recommendations, or other feedback or information (collectively, “<b>Feedback</b>”), User hereby grants to Company a fully paid-up, royalty free, worldwide, perpetual, irrevocable, transferable, sub-licensable license to freely use, disclose, and otherwise exploit such Feedback, including but not limited to incorporation of the Feedback into future versions of the Storage Services without any compensation or other obligation to User. </p>
```
Comment (boshevski) [inline content/terms-of-service/termsOfService.md]: Add space or new line after the title "10. Feedback"

Comment (mbouzi) [reply content/terms-of-service/termsOfService.md]: Screen Shot 2020-03-27 at 4 34 38 PM

File: content/terms-of-service/termsOfService.md (hunk 2 above)
Comment (mbouzi) [reply content/terms-of-service/termsOfService.md:12]: Screen Shot 2020-03-27 at 4 34 47 PM

**Pull request merged**